### PR TITLE
enh: rearranges tabs in order and attendee view

### LINKED ISSUE #3233: Make Orders and Attendee View in Ticket Tab Logical
The orders and attendee tab show "All" orders immediately. This is not necessary as the organizer usually cares less about expired tickets. They want to see the orders that are completed and placed.

Therefore change the orders and attendee tab as follows:
* Show "Completed" in front
* Move "All" to the last place
* Take out "Pending" from the organizer view. It is not relevant for them.

Screenshot from 2019-07-03 22-19-48

### PR DESCRIPTION
**Pull request by shreyanshdwivedi**
Fixes #3233

#### Short description of what this resolves:
The orders and attendee tab show "All" orders immediately. This is not necessary 

#### Changes proposed in this pull request:
- Show "Completed" in front
- Move "All" to the last place
- Take out "Pending" from the organizer view. It is not relevant for them.

**Admin Access**
 - _Orders_
Screenshot_2019-07-04 Orders Tickets AddOwnerEvent Events Open Event

 - _Attendees_
Screenshot_2019-07-04 Attendees Tickets AddOwnerEvent Events Open Event

**Non Admin (Organizer)**
 - _Orders_
Screenshot_2019-07-04 Orders Tickets AddOwnerEvent Events Open Event(1)

 - _Attendees_
Screenshot_2019-07-04 Attendees Tickets AddOwnerEvent Events Open Event(1)

#### Checklist

- [x] I have read the [Contribution & Best practices Guide](https://blog.fossasia.org/open-source-developer-guide-and-best-practices-at-fossasia).
- [x] My branch is up-to-date with the Upstream `development` branch.
- [x] The acceptance, integration, unit tests and linter pass locally with my changes 
- [ ] I have added tests that prove my fix is effective or that my feature works
- [ ] I have added necessary documentation (if appropriate)

### CODE REVIEW

**Comment by shreyanshdwivedi** [comment]
@uds5501 @CosmicCoder96 @kushthedude please review

File: app/controllers/events/view/tickets/attendees.js [hunk 1]
```diff
@@ -1,11 +1,16 @@
...
 import { run } from '@ember/runloop';
+import { computed } from '@ember/object';
 
 export default Controller.extend({
   isLoadingcsv: false,
 
   isLoadingpdf: false,
 
+  showPendingRoute: computed('authManager.currentUser.isAdmin', 'authManager.currentUser.isSuperAdmin', function() {
+    return this.get('authManager.currentUser.isSuperAdmin') || this.get('authManager.currentUser.isAdmin');
```
Comment (kushthedude) [inline app/controllers/events/view/tickets/attendees.js]: Please use [`or`](http://api.emberjs.com/ember/3.10/functions/@ember%2Fobject%2Fcomputed/or)

File: app/controllers/events/view/tickets/orders.js [hunk 2]
```diff
@@ -1,11 +1,16 @@
...
 import { run } from '@ember/runloop';
+import { computed } from '@ember/object';
 
 export default Controller.extend({
   isLoadingcsv: false,
 
   isLoadingpdf: false,
 
+  showPendingRoute: computed('authManager.currentUser.isAdmin', 'authManager.currentUser.isSuperAdmin', function() {
+    return this.get('authManager.currentUser.isSuperAdmin') || this.get('authManager.currentUser.isAdmin');
```
Comment (kushthedude) [inline app/controllers/events/view/tickets/orders.js]: Please use [`or`](http://api.emberjs.com/ember/3.10/functions/@ember%2Fobject%2Fcomputed/or)

**Comment by shreyanshdwivedi** [comment]
@kushthedude @pradeepgangwar @uds5501 I've updated the logic. Please review.

**Pull request merged**